docker-compose.production.yml

```diff
@@ -14,7 +14,7 @@ secrets:
 
 services:
   app:
-    image: vilnius/tvarkauvilniums:production
+    image: kurmisrainas/tvarkau-vilniu-ms
```

Comment:
this is ok?

Reply:
yes, ok for now - it is built from my own branch. we can setup proper account later